jobwatcher/jobwatcher.py

```diff
@@ -189,7 +190,11 @@ def _get_config():
         "Configured parameters: region=%s scheduler=%s stack_name=%s instance_type=%s pcluster_dir=%s proxy=%s",
         region, scheduler, stack_name, instance_type, pcluster_dir, _proxy
     )
-    return region, scheduler, stack_name, instance_type, pcluster_dir, proxy_config
+
+    JobwatcherConfig = collections.namedtuple(
```

Comment:
since this is like defining a class we could also have it defined outside of the function scope

Reply:
Done